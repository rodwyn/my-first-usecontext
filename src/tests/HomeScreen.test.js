import { mount } from 'enzyme';
import React from 'react';
import { HomeScreen } from '../HomeScreen';
import { UserContext } from '../UserContext';

describe('Testing <HomeScreen /> component.', () => {
  const user = {
    id: '007',
    name: 'Rodwyn Moreno',
    email: 'rodwynmoreno@gmail.com'
  };
  const wrapper = mount(
    <UserContext.Provider value={{
      user
    }}>
      <HomeScreen />
    </UserContext.Provider>
  
  );

  test('should render correctly.', () => {
    expect(wrapper).toMatchSnapshot();
  });
});