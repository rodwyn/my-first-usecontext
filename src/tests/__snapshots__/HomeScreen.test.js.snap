// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing <HomeScreen /> component. should render correctly. 1`] = `
<div>
  <h1>
    HomeScreen
  </h1>
  <hr />
  <pre>
    {
  "id": "007",
  "name": "Rodwyn Moreno",
  "email": "rodwynmoreno@gmail.com"
}
  </pre>
</div>
`;
